README.md

```diff
@@ -258,7 +260,11 @@ Each time you want to have your terminal environment setup you will want to run
 
 | Command | Description |
 |--------|-------------|
+| `analyze` | Runs `composer analyze` and `vendor/bin/phpstan analyze` inside `platform.test` container |
 | `artisan <command>` | Runs any Laravel Artisan command inside `platform.test` container |
+| `comp <command>` | Runs any Composer command inside `platform.test` container |
+| `comp <command>` | Runs any Composer command inside `platform.test` container |
```

Comment:
It looks like this is duplicated. Did you mean to add a different command here?

If you're looking for others to add, other composer commands I typically run are:
- `composer format`
- `composer localize`

Reply:
I've added those aliases and removed the duplicate line in the readme. 

Those commands since they operate on the code it's best to run them locally instead of the container like most. 

Dependending on the volume mirroring from your dockers I wouldn't want to guarantee they would properly update all the local files and have proper ownership and permissions.

Composer and PHP8.4 are installed when you run `nix-shell` so it will be in your path locally if the aliases are.

If you see any others that should similarly be run locally instead of containers let me know?